src/networking.c

```diff
@@ -3701,75 +3767,19 @@ void clientCommand(client *c) {
     } else if (!strcasecmp(c->argv[1]->ptr, "kill")) {
         /* CLIENT KILL <ip:port>
          * CLIENT KILL <option> [value] ... <option> [value] */
-        char *addr = NULL;
-        char *laddr = NULL;
-        user *user = NULL;
-        int type = -1;
-        uint64_t id = 0;
-        long long max_age = 0;
-        int skipme = 1;
+        clientFilter client_filter = {0, 0, NULL, NULL, NULL, -1, 1};
```

Comment:
Can you write the struct with the field names, so it's easier to read it?

```suggestion
        clientFilter client_filter = {
            .type = -1,
            .skipme = 1,
        };
```

Reply:
Good. Just FYI, it's possible to skip fields that are 0 or NULL, because they are zeroed by default with this syntax.

Reply:
TIL!